controllers/logout_test.go

```diff
@@ -27,7 +32,14 @@ func TestLogoutTestSuit(t *testing.T) {
 }
 
 func (s *LogoutTestSuite) TestLogoutWithAuthenticatedUser() {
-	cookie := fixture.GenerateCookie("user_id", "test-user")
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte("testPassword"), bcrypt.DefaultCost)
+	if err != nil {
+		log.Fatalf("Cannot hash password: %s", err)
+	}
+	user := models.User{Email: "test@email.com", Password: string(hashedPassword)}
+	db.GetDB().Create(&user)
```

Comment:
@gutakk How about moving this to a helper function? I think we can put it in `fixture` 🤔

Reply:
I will refactor this in the refactor test [PR](https://github.com/gutakk/go-google-scraper/issues/36) 🙏